tools/pipelines/templates/build-npm-package.yml

```diff
@@ -642,21 +639,29 @@ extends:
                   officeFeedUrl: $(ado-feeds-office)
                   isCheckoutNeeded: true
               - task: Bash@3
-                displayName: Retrieve buildId results
+                displayName: Get stage timing and result data from ADO
+                env:
+                  BUILD_ID: $(Build.BuildId)
+                  ADO_API_TOKEN: $(System.AccessToken)
+                  WORK_FOLDER: $(pipelineTelemetryWorkdir)
                 inputs:
                   targetType: inline
-                  workingDirectory: $(absolutePathToTelemetryGenerator)
```

Comment:
Now that the task is split into "get data" and "send data to Kusto", the "get data" piece doesn't need to run in this directory.